apps/src/lib/kits/maker/ui/SetupChecklist.jsx

```diff
@@ -365,20 +365,10 @@ class SetupChecklist extends Component {
       firmataMarkdown = applabI18n.makerSetupInstallFirmataMB({
         firmataURL: MICROBIT_FIRMATA_URL,
       });
-    } else if (
-      this.state.boardTypeDetected === BOARD_TYPE.EXPRESS ||
-      this.state.boardTypeDetected === BOARD_TYPE.CLASSIC
-    ) {
-      firmataMarkdown = applabI18n.makerSetupInstallFirmataCP({
-        firmataURLExpress: EXPRESS_FIRMATA_URL,
-        firmataURLClassic: CLASSIC_FIRMATA_URL,
-      });
     } else {
-      // Board Type is Other/Unknown
```

Comment:
This is unnecessary since we filter ports so that we only accept serial devices if they're either micro:bit, circuit playground express, or circuit playground classic [here](https://github.com/code-dot-org/code-dot-org/blob/4044721288e195fb88d6c5db5c5b38445a1cb6c2/apps/src/lib/kits/maker/ui/SetupInstructions.jsx#L48).